nodejs/nodejs-hello-world/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/nodejs-hello-world.iml" filepath="$PROJECT_DIR$/.idea/nodejs-hello-world.iml" />
```

Comment:
Are both parameters needed?

Reply:
You mean “fileurl” and “filepath”? I’m not totally sure, but these files were auto generated by the IDE so I lean towards leaving them as is.

Reply:
Yeah, if it's auto-generated, let's leave them. I'm merging your PR.

Thanks!